examples/natmod/stm32wb55_transparent_vcp/_stm32wb55_transparent.c

```diff
@@ -0,0 +1,639 @@
+#include <string.h>
+#include "py/dynruntime.h"
+#include "py/stream.h"
+#include "stm32wbxx_ll_utils.h"
+#include "stm32wbxx_ll_system.h"
+#include "ports/stm32/mpbthciport.h"
+#include "ports/stm32/rfcore.h"
```

Comment:
I don't think these 2 includes are needed, neither the `py/stream.h` one.  Then a lot of the `CFLAGS` lines in the Makefile can also be removed.

Reply:
Ah yes, in earlier versions of this I was building `stm32/rfcore.c` into the module as well to use some of its functions... until I realised I obviously couldn't access the shared memory buffers this way.

Reply:
Yep latest push has all of these cleaned up / minimsed and as suggested, the Makefile was able to become much simpler as well.